app/views/estimate_flow/benefits_more.html.slim

```diff
@@ -0,0 +1,25 @@
+- content_for :back
+  = link_to t("generic.back"), wizard_path(previous_step_for(@estimate, :benefit_yesno)), class: "govuk-back-link"
+
+.govuk-grid-column-two-thirds
+  span.govuk-caption-l = t(".caption")
+  h1.govuk-heading-l = t(".heading.text", count: @form.benefits.size)
+
+  = govuk_table do |table|
+    - table.body do |body|
+      - @form.benefits.each do |item|
+        - body.row do |row|
+          - row.cell(text: item.fetch("benefit_type"))
+          - row.cell(text: item.fetch("benefit_amount"))
+          - row.cell(text: item.fetch("benefit_frequency"))
```

Comment:
Is this PR deliberately leaving out the change/remove links from here?

Reply:
oops yes forgot that bit